instrumentation/elasticsearch/elasticsearch-transport-5.3/javaagent/build.gradle.kts

```diff
@@ -1,59 +1,59 @@
 plugins {
-  id("otel.javaagent-instrumentation")
+    id("otel.javaagent-instrumentation")
```

Comment:
You might want to revert formatting changes in this file (and a few others in this PR)